scripts/test/watch

```diff
@@ -1,12 +0,0 @@
-#!/bin/sh
-
-set -e
-
-filewatcher \
-    -L 5 \
-    -x '**/*.swp' \
-    -x .git \
-    -x build \
-    -x .idea \
-    -- \
-    sh -c 'go test -timeout 10s -v ./${dir} || ( echo; echo; exit 1 )'
```

Comment:
The makefile builds the artifacts.

Different developers have different ways of launching unit tests. For example, I have `<leader>t` launch my unit tests. This doesn't require any support from the build system. If you want to launch them on a watch, then make a local alias that sets that up. No need to add a dependency and more stuff into the build system that are specific to a workflow.